docs/product/performance/mobile-vitals/app-starts.mdx

```diff
@@ -104,4 +104,4 @@ Sentry automatically identifies sample events to help you investigate performanc
 
 You can use the sample list to drill down to and compare fast, average, and slow events of interest within a given screen.
 
-Clicking on a sample event will take you into either the query’s span details within the span waterfall of the **Event Details** page or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
+Clicking on a sample event will take you into either the query’s span details within the span waterfall of the [Trace View page](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
```

Comment:
Looks like the page has been renamed to just "Trace" not "Trace View". Maybe something we want to update?